content/SCALE/SCALECLIReference/Auth/CLISessions.md

```diff
@@ -12,4 +12,225 @@ tags:
 
 {{< include file="/_includes/CLIGuideWIP.md" type="page" >}}
 
+## Sessions Commands
+
+The **sessions** namespace has seven commands and is based on functions found in the SCALE API and web UI. 
+It provides access to session information through the seven **sessions** commands. 
+
+You can enter commands from the main CLI prompt or the **sessions** namespace prompt.
+
+### Sessions Command
+
+The `sessions` command returns a table that displays each session id, whether or not each session is current and internal, each session origin, and the credentials and credential data used to log into each session.
+
+{{< expand "Displaying all sessions since system boot" "v" >}}
+#### Description
+The `sessions` command does not require entering property or arguments.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table with the ID, current session state, origin such as an IP or socket information, credentials, and creation date and time.
+Enter the command, then press <kbd>Enter</kbd>.
+The command returns a table with the ID, current session state, origin such as an IP or socket information, credentials, and creation date and time.
+
+#### Usage
+
+From the CLI Auth namespace prompt, enter:
+
+`sessions`
+
+{{< expand "Command Example" "v" >}}
+```
+auth> sessions
++--------------------------------------+---------+----------+---------------------------------------+-------------+------------------+---------------------------+
+| id                                   | current | internal | origin                                | credentials | credentials_data | created_at                |
++--------------------------------------+---------+----------+---------------------------------------+-------------+------------------+---------------------------+
+| 8a21fefe-1255-42eb-a86d-8a7f74752b94 | false   | false    | 10.234.15.210:15864                   | UNIX_SOCKET | <dict>           | 2023-07-25T17:54:49+00:00 |
+| f01274e8-4954-4926-9346-76a16d3648dd | true    | true     | UNIX socket (pid=2459768 uid=0 gid=0) | UNIX_SOCKET | <dict>           | 2023-07-26T11:05:16+00:00 |
++--------------------------------------+---------+----------+---------------------------------------+-------------+------------------+---------------------------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### ID Command
+
+The `id` command displays the IDs of all TrueNAS sessions since boot. 
+
+{{< expand "Identifying sessions" "v" >}}
+The `id` command has no required options. It returns a table of recent and active session IDs.
+
```

Comment:
Not following the template in this section.
Add the line about entering the command and the result.